test/unit/helpers/xhr/MFA_CHALLENGE_push_rejected_app_upgrade_needed.js

```diff
@@ -0,0 +1,77 @@
+define({
+    "status": 200,
+    "responseType": "json",
+    "response": {
+      "stateToken": "testStateToken",
+      "expiresAt": "2015-10-22T02:23:28.893Z",
+      "status": "MFA_CHALLENGE",
+      "factorResult": "REJECTED",
+      "factorResultMessage": "OKTA_VERIFY_UPGRADE_REQUIRED",
```

Comment:
if this is the only difference with `MFA_CHALLENGE_push_rejected` file, maybe instead of adding a new file we can clone the object in the test and add this? e.g.
```
const resRejectedPushAppUpgrade = JSON.parse(JSON.stringify(resRejectedPush));
resRejectedPushAppUpgrade.repsonse.factorResultMessage = 'OKTA_VERIFY_UPGRADE_REQUIRED';
```